src/modules/ekf2/EKF/aux_global_position.cpp

```diff
@@ -119,6 +122,13 @@ void AuxGlobalPosition::update(Ekf &ekf, const estimator::imuSample &imu_delayed
 			if (continuing_conditions) {
 				ekf.fuseHorizontalPosition(aid_src);
 
+				if (isTimedOut(aid_src.time_last_fuse, imu_delayed.time_us, ekf._params.no_aid_timeout_max)
```

Comment:
For safety it might be better if this respects the existence of other healthy horizontal aiding sources or perhaps GPS specifically.

Reply:
Yes, let me do that.